theme/static/style/color-palette-light.css

```diff
@@ -22,6 +22,17 @@ nav.primary a.active, #pagination a.active {
     color: #e3edea;
 }
 
+.mobilenav ul a:hover{
+    background-color: #1c3d32;
+    color: #fff;
+}
```

Comment:
I don't think this definition does anything useful b/c on mobile we can't really hover above the links. Please explain why we need it or remove it.